src/main/resources/assets/schemas/DataAccessRequest.yaml

```diff
@@ -203,3 +203,28 @@ properties:
       including any data that are downloaded or utilized in the cloud.
     items:
       $ref: './Collaborator.yaml'
+  progressReportSummary:
+    type: string
+    description: A Progress Report Summary.
+  intellectualPropertySummary:
+    type: string
+    description: An Intellectual Property Summary.
+  publications:
+    type: array
+    description: An array of publications.
+    items:
+      $ref: './Publication.yaml'
+  presentations:
+    type: array
+    description: An array of presentations.
+    items:
+      $ref: './Presentation.yaml'
+  dataManagementIncident:
+    $ref: './DataManagementIncident.yaml'
+    description: A Data Management Incident.
```

Comment:
There is a warning here that is a little confusing at first. The root cause is that if there is a `$ref`, then anything after it, i.e. description, is ignored. The idea is that all information should in theory be inside the `$ref`, not here in the property. Same thing applies to `closeOutSupplement`

Reply:
Example of an incorrect intellij warning:

![Screenshot 2025-01-21 at 7 48 07 AM](https://github.com/user-attachments/assets/a67f040a-0c2f-42aa-84c4-5acb2c2864cb)